content/en/prio.html

```diff
@@ -8,16 +8,20 @@
     <div class="container">
         <h2>What is Prio?</h2>
         <p></p>
-        <p>Prio is a technology that technically enforces the privacy of application metrics.</p>
+        <p><a href=https://crypto.stanford.edu/prio/>Prio</a> is a privacy-respecting system for the collection of aggregate statistics such as application metrics.</p>
+
+        <p>In Prio, applications such as a web browser, mobile application or a website generate data, split it into two anonymized and encrypted shares shares, and upload each share to a different data share processor, such that neither processor can learn much about the original data. Each processor then aggregates its data shares into a partial sum, which can be combined into the final aggregation, permitting useful statistics over the whole body of data to be computed while revealing barely any information about individual participants, so long as either one of the data share processors does not disclose their data shares.</p>
```

Comment:
From the first sentence: "split it into two anonymized and encrypted shares shares"

It's not clear to me what "it" refers to here (just write it out) and the word "shares" is repeated twice in a row.